From b74564f97c395bb2ce32731481813fa1b192adfe Mon Sep 17 00:00:00 2001
From: Pablo Saavedra <psaavedra@igalia.com>
Date: Mon, 11 Apr 2022 11:07:41 +0200
Subject: [PATCH] [WPE] Build ANGLE with the EGL_NO_PLATFORM_SPECIFIC_TYPES
 define https://bugs.webkit.org/show_bug.cgi?id=239039

Specify the EGL_NO_PLATFORM_SPECIFIC_TYPES define when building ANGLE
subproject for the GTK and WPE ports. This should avoid searching for
platform-specific headers that might not be available at all during
build, e.g. the X11 headers which are used by default on UNIX platforms.
---
 Source/ThirdParty/ANGLE/PlatformWPE.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/ThirdParty/ANGLE/PlatformWPE.cmake b/Source/ThirdParty/ANGLE/PlatformWPE.cmake
index a4d0b959..8f5bff7f 100644
--- a/Source/ThirdParty/ANGLE/PlatformWPE.cmake
+++ b/Source/ThirdParty/ANGLE/PlatformWPE.cmake
@@ -1,4 +1,4 @@
-list(APPEND ANGLE_DEFINITIONS ANGLE_PLATFORM_LINUX)
+list(APPEND ANGLE_DEFINITIONS ANGLE_PLATFORM_LINUX EGL_NO_PLATFORM_SPECIFIC_TYPES)
 include(linux.cmake)
 
 if (USE_OPENGL)
